agent/agent-tooling/src/main/java/com/microsoft/applicationinsights/agent/internal/wasbootstrap/configuration/ConfigurationBuilder.java

```diff
@@ -200,7 +200,7 @@ public static void overlayEnvVars(Configuration config) throws IOException {
 
         if (isTrimEmpty(config.role.name)) {
             // only use WEBSITE_SITE_NAME as a fallback
```

Comment:
comment can be deleted too?

Reply:
I think the comment is still good, in other cases env var overrides the json config, but in this case it's only a fallback